responsive-style.css

```diff
@@ -0,0 +1,237 @@
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .main {
+    background-image: url(./img/Header\ bg.svg);
+    background-size: cover;
+
+    /* height: 916px; */
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .logo {
+    padding-left: 143px;
+  }
+
+  .nav-items {
+    padding-right: 142px;
+  }
+
+  .nav-items a {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+    padding: 0 12px;
+  }
+
+  .primary-text,
+  .social,
+  .social-media-icons {
+    padding-left: 260px;
+  }
+
+  .content {
+    padding: 0 260px;
+  }
+
+  .container {
+    display: grid;
+    column-gap: 0;
+    row-gap: 88px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    width: 60%;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 0;
+    margin: 50px 100px;
+    height: fit-content;
+  }
+
+  .snapshot-details {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 544px;
+    height: fit-content;
+    align-items: flex-start;
+  }
+
+  .snapshot {
+    width: 554px;
+    border-radius: 1px;
+  }
+
```

Comment:
- [ ] As stated the project requirement You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly make sure that work section is responsive for  desktop screen size(between 768 and 1440), Please adjust the width.

Your Project Work section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206440179-d62af892-d5f4-4cdd-a6da-38b042f7be1c.PNG)

The Figma Design Work section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206440684-dcb9dca6-85bc-448c-9ad3-1272869006b6.PNG)